assets/js/blocks/products/all-products/edit.js

```diff
@@ -148,7 +148,10 @@ class Editor extends Component {
 					controls={ [
 						{
 							icon: 'edit',
-							title: __( 'Edit', 'woo-gutenberg-products-block' ),
+							title: __(
+								'Rearrange inner blocks',
```

Comment:
We can not only Rearrange but add new/remove blocks for inner items, so I think this should be something like `Edit inner product layout`.